packaging/export/export_kbfs.sh

```diff
@@ -46,5 +46,5 @@ rm $src_dir/$name.tar
 cd $dest_dir
 echo "Committing"
 git add .
-git commit -m "Updating repo"
+git commit -m "Updating repo" --allow-empty
```

Comment:
I could also make this `git commit ... || echo "no changes -- not committing"`, if we prefer to avoid putting crap in the history.